diff -Naru dialog/scripts/config/conf.c dialog-patched/scripts/config/conf.c
--- dialog/scripts/config/conf.c	2006-07-01 01:42:01.000000000 +0300
+++ dialog-patched/scripts/config/conf.c	2007-02-22 19:16:53.000000000 +0200
@@ -546,9 +546,9 @@
 		}
 		break;
 	case ask_silent:
-		if (stat(".config", &tmpstat)) {
+		if (stat("user_batch.conf", &tmpstat)) {
 			printf("***\n"
-				"*** You have not yet configured BusyBox!\n"
+				"*** You have not yet configured curl-loader!\n"
 				"***\n"
 				"*** Please run some configurator (e.g. \"make oldconfig\" or\n"
 				"*** \"make menuconfig\" or \"make config\").\n"
@@ -576,7 +576,7 @@
 		check_conf(&rootmenu);
 	} while (conf_cnt);
 	if (conf_write(NULL)) {
-		fprintf(stderr, "\n*** Error during writing of the BusyBox configuration.\n\n");
+		fprintf(stderr, "\n*** Error during writing of the curl-loader configuration.\n\n");
 		return 1;
 	}
 	return 0;
diff -Naru dialog/scripts/config/confdata.c dialog-patched/scripts/config/confdata.c
--- dialog/scripts/config/confdata.c	2006-07-01 01:42:01.000000000 +0300
+++ dialog-patched/scripts/config/confdata.c	2007-02-22 17:39:38.000000000 +0200
@@ -13,12 +13,12 @@
 #define LKC_DIRECT_LINK
 #include "lkc.h"
 
-const char conf_def_filename[] = ".config";
+const char conf_def_filename[] = "user_batch.conf";
 
 const char conf_defname[] = "defconfig";
 
 const char *conf_confnames[] = {
-	".config",
+	"user_batch.conf",
 	conf_defname,
 	NULL,
 };
diff -Naru dialog/scripts/config/mconf.c dialog-patched/scripts/config/mconf.c
--- dialog/scripts/config/mconf.c	2006-07-01 01:42:01.000000000 +0300
+++ dialog-patched/scripts/config/mconf.c	2007-02-22 19:17:20.000000000 +0200
@@ -31,7 +31,7 @@
 static const char mconf_readme[] =
 "Overview\n"
 "--------\n"
-"Some features may be built directly into BusyBox.  Some features\n"
+"Some features may be built directly into curl-loader.  Some features\n"
 "may be completely removed altogether.  There are also certain\n"
 "parameters which are not really features, but must be\n"
 "entered in as decimal or hexadecimal numbers or possibly text.\n"
@@ -125,7 +125,7 @@
 "Even if you don't use alternate configuration files, but you\n"
 "find during a Menuconfig session that you have completely messed\n"
 "up your settings, you may use the \"Load Alternate...\" option to\n"
-"restore your previously saved settings from \".config\" without\n"
+"restore your previously saved settings from \"user_batch.conf\" without\n"
 "restarting Menuconfig.\n"
 "\n"
 "Other information\n"
@@ -193,11 +193,11 @@
 	"last retrieved.  Leave blank to abort.",
 load_config_help[] =
 	"\n"
-	"For various reasons, one may wish to keep several different BusyBox\n"
+	"For various reasons, one may wish to keep several different curl-loader\n"
 	"configurations available on a single machine.\n"
 	"\n"
 	"If you have saved a previous configuration in a file other than the\n"
-	"BusyBox's default, entering the name of the file here will allow you\n"
+	"curl-loader's default, entering the name of the file here will allow you\n"
 	"to modify that configuration.\n"
 	"\n"
 	"If you are uncertain, then you have probably never used alternate\n"
@@ -207,7 +207,7 @@
 	"as an alternate.  Leave blank to abort.",
 save_config_help[] =
 	"\n"
-	"For various reasons, one may wish to keep different BusyBox\n"
+	"For various reasons, one may wish to keep different curl-loader\n"
 	"configurations available on a single machine.\n"
 	"\n"
 	"Entering a file name here will allow you to later retrieve, modify\n"
@@ -255,7 +255,7 @@
 	"          USB$ => find all CONFIG_ symbols ending with USB\n"
 	"\n";
 
-static char filename[PATH_MAX+1] = ".config";
+static char filename[PATH_MAX+1] = "user_batch.conf";
 static int indent;
 static struct termios ios_org;
 static int rows = 0, cols = 0;
@@ -939,7 +939,7 @@
 
 	sym = sym_lookup("VERSION", 0);
 	sym_calc_value(sym);
-	snprintf(menu_backtitle, 128, "BusyBox v%s Configuration",
+	snprintf(menu_backtitle, 128, "curl-loader v%s Configuration",
 		sym_get_string_value(sym));
 
 	mode = getenv("MENUCONFIG_MODE");
@@ -960,16 +960,16 @@
 	init_dialog();
 	do {
 		stat = dialog_yesno(NULL,
-				    "Do you wish to save your new BusyBox configuration?", 5, 60);
+				    "Do you wish to save your new curl-loader configuration?", 5, 60);
 	} while (stat < 0);
 	end_dialog();
 
 	if (stat == 0) {
 		conf_write(NULL);
 		printf("\n"
-			"*** End of BusyBox configuration.\n");
+			"*** End of curl-loader configuration.\n");
 	} else
-		printf("\n\nYour BusyBox configuration changes were NOT saved.\n\n");
+		printf("\n\nYour curl-loader configuration changes were NOT saved.\n\n");
 
 	return 0;
 }
diff -Naru dialog/scripts/config/zconf.tab.c_shipped dialog-patched/scripts/config/zconf.tab.c_shipped
--- dialog/scripts/config/zconf.tab.c_shipped	2006-07-01 01:42:01.000000000 +0300
+++ dialog-patched/scripts/config/zconf.tab.c_shipped	2007-02-22 19:17:54.000000000 +0200
@@ -1925,7 +1925,7 @@
 	sym_init();
 	menu_init();
 	modules_sym = sym_lookup("MODULES", 0);
-	rootmenu.prompt = menu_add_prop(P_MENU, "BusyBox Configuration", NULL, NULL);
+	rootmenu.prompt = menu_add_prop(P_MENU, "Curl-loader Configuration", NULL, NULL);
 
 	//zconfdebug = 1;
 	zconfparse();
diff -Naru dialog/scripts/config/zconf.y dialog-patched/scripts/config/zconf.y
--- dialog/scripts/config/zconf.y	2006-07-01 01:42:01.000000000 +0300
+++ dialog-patched/scripts/config/zconf.y	2007-02-22 19:18:08.000000000 +0200
@@ -487,7 +487,7 @@
 	sym_init();
 	menu_init();
 	modules_sym = sym_lookup("MODULES", 0);
-	rootmenu.prompt = menu_add_prop(P_MENU, "BusyBox Configuration", NULL, NULL);
+	rootmenu.prompt = menu_add_prop(P_MENU, "Curl-loader Configuration", NULL, NULL);
 
 	//zconfdebug = 1;
 	zconfparse();
